packages/node_modules/@webex/plugin-meetings/src/common/collection.js

```diff
@@ -0,0 +1,75 @@
+import {MEETINGS} from '../constants';
+
+/**
+ * @class Collection
+ */
+export default class Collection {
+  namespace = MEETINGS;
+
+  /**
+   * @param {String} property
+   * @memberof Collection
+   */
+  constructor(property) {
+    this.propertyName = property;
+    this[this.propertyName] = {};
+  }
+
+  /**
+   * @param {String} id ID of the thing stuffed into the collection at id location
+   * @returns {Any} returns whatever is being stuffed into the collection
+   * @public
+   * @memberof Collection
+   */
+  get(id) {
+    if (this[this.propertyName] && this[this.propertyName][id]) {
+      return this[this.propertyName][id];
+    }
+    return null;
+  }
+
+  /**
+   * @param {String} id the id of the meeting info instance to add to the collection
+   * @param {Any} set the thing to set in the collection
+   * @returns {Any} returns the thing just put in the collection
+   * @public
+   * @memberof Collection
+   */
+  set(id, set) {
+    this[this.propertyName][id] = set;
+    return this.get(id);
+  }
+
+  /**
+   * remove the thing at the id
+   * @param {String} id ID of the thing you wish to delete from the collection
+   * @returns {Any} returns the thing that was removed
+   * @public
+   * @memberof Collection
+   */
+  delete(id) {
+    let deleted = this[this.propertyName][id];
+    delete this[this.propertyName][id];
+    deleted = null;
+    return deleted;
+  }
+
+  /**
+   * @returns {Object} returns an object map of things stuffed into the collection
+   * @public
+   * @memberof Collection
+   */
+  getAll() {
+    return this.meetingInfos;
```

Comment:
Will this always be `meetingInfos`? Or could it be generic?

Reply:
oops, that should be this[this.propertyName]